config/manifests/bases/idp-mgmt-operator.clusterserviceversion.yaml

```diff
@@ -6,6 +6,9 @@ metadata:
     capabilities: Basic Install
     categories: Integration & Delivery
     operatorframework.io/suggested-namespace: idp-mgmt-config
+    operatorframework.io/initialization-resource: '{"apiVersion":"identityconfig.identitatem.io/v1alpha1",
+       "kind":"IDPConfig","metadata":{"name":"idp-config","namespace":"idp-mgmt-config"},
+       "spec": {}}'    
```

Comment:
Glad this is added because it will really show the user there is more to do after they click Install.  So even if they ignore the doc, this will be a reminder!